source/xml/reading.md

```diff
@@ -376,6 +377,7 @@ their respective XML elements:
             }
             return $books;
         },
+        // handle a single book
         '{http://example.org/books}book' => function($reader) {
             $book = new Book();
```

Comment:
question, not related to the PR:

do we really need to read the content into a KeyValue and manually turn it into a object in a second step?
wouldnt it be easier to have a deserializer which uses a given object, like here `Book`, and populates all its properties (so we get a regular value object)?